src/app/smoothie/conference/conference.component.jsx

```diff
@@ -1,22 +1,78 @@
 import React from 'react';
 import PropType from 'prop-types';
 import './conference.css';
+import Modal from './conferenceModal.component';
 
-const Conference = ({ name, timeBegin, timeEnd }) => (
-  <div className="columns">
-    <div className="column">
-      <div className="conference">
-        <div className="conference-opt"><a><i className="fa fa-arrows-h circle" /></a><span className="conference-time">{`${timeBegin}h00 > ${timeEnd}h00`}</span><a><i className="fa fa-lock circle" /></a></div>
-        <div className="conference-title">{name !== undefined ? name : 'Temps libre'}</div>
+class Conference extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalState: false,
+    };
+
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  toggleModal() {
+    this.setState((prev) => {
+      const newState = !prev.modalState;
+      return {
+        modalState: newState,
+      };
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <Modal
+          closeModal={this.toggleModal}
+          modalState={this.state.modalState}
+          title={this.props.name !== undefined ? this.props.name : 'Temps libre'}
+          speaker={this.props.speaker}
```

Comment:
tu dois pouvoir passer tes props via un {...this.props} plutôt que une par une ^^